lib/bundle-manager.js

```diff
@@ -140,6 +146,11 @@ module.exports.init = function (bundlesPath, cfgPath, nodecgVersion, nodecgConfi
 			return;
 		}
 
+		// Prevent attempting to load unwanted directories. Those specified above and all dot-prefixed.
+		if (blacklistedBundleDirectories.indexOf(bundleFolderName) > -1 || bundleFolderName.startsWith('.')) {
```

Comment:
`blacklistedBundleDirectories.indexOf(bundleFolderName) > -1` can be simplified to `blacklistedBundleDirectories.includes(bundleFolderName)`